gradle/libs.versions.toml

```diff
@@ -1,8 +1,8 @@
 [versions]
-nva = { strictly = "1.7.7" }
+nva = { strictly = "1.8.1" }
 nvaDatamodelVersion = { strictly = "0.13.1" }
 jacksonVersion = { strictly = "2.12.4" }
-awsSdkVersion = { strictly = "1.12.42" }
+awsSdkVersion = { strictly = "1.12.52" }
```

Comment:
had to update to the same version as the one used in nva-commons. Strictly imposes the same version everywhere.